src/extension/content.js

```diff
@@ -15,10 +15,33 @@
  * @typedef {import('@Types').OptionsConfig} OptionsConfig
  */
 
+/**
+ * Removes the cache buster from the URL.
+ */
+function removeCacheParam() {
```

Comment:
Don't love this defined here but didn't think it fit into any other util files and didn't want to create another one (to keep the number of files the sidekick loads to a minimum). Do you see this fitting anywhere else?

Reply:
I don't particularly mind having this here, but maybe `tab.js` would be a better fit. We could call it from `checkTab()`.

Reply:
Since `removeCacheParam()` uses `window` we can't call it from `checkTab` unfortunately. Since you don't mind, I'm thinking I will just keep it where it is.